configure.ac

```diff
@@ -267,6 +267,7 @@ AM_CONDITIONAL([MAKE_TESTS], [test x$have_check = xyes])
 
 PKG_CHECK_MODULES([GLIB], [glib-2.0 >= 2.16],[],[AC_MSG_ERROR(GLib 2.16 or newer not found. Install missing package and re-run)])
 PKG_CHECK_MODULES([JSONC], [json-c], [have_jsonc='yes'], [have_jsonc='no'])
+PKG_CHECK_MODULES([ZLIB], [zlib], [], [AC_MSG_ERROR([zlib not found])])
```

Comment:
I don't see why ZLIB is being added as a dependency, where is it being used?

Reply:
It is used to inflate compressed cluster in `vbd_qcow2_uncompress_cluster` (`libvmi/disk/vbd.c`).

Reply:
I see. Please make the zlib dependency optional, if its not installed the qcow2 parsing should just be disabled. Thanks